src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java

```diff
@@ -73,4 +73,23 @@ public void mercury_navigation_isFooterAlwaysVisible() {
     driver.executeScript("window.scrollTo(100, document.body.scrollHeight)");
     Assertion.assertTrue(navigation.isFooterVisible());
   }
+
+  @Test(groups = "mercury_navigation_navigationOnEnglishWiki")
+  public void mercury_navigation_navigationOnEnglishWiki() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)